DataFormats/TotemReco/src/classes_def.xml

```diff
@@ -7,4 +7,12 @@
   <class name="edm::Wrapper<edm::DetSetVector<TotemT2Digi> >"/>
   <class name="std::vector<TotemT2Digi>"/>
   <class name="std::vector<edm::DetSet<TotemT2Digi> >"/>
+  <class name="TotemT2RecHit" ClassVersion="2">
+    <version ClassVersion="2" checksum="969691996"/>
```

Comment:
Class version may be bumped to 10 in a follow-up push? Not sure what is the current convention nowadays.